test/tests.js

```diff
@@ -230,6 +230,88 @@ test('dynamic queue', function(assert) {
   });
 });
 
+test('state machine', function(assert) {
```

Comment:
I am scratching my head why houndci-bot flagged this. The linter didn't have a problem with it.